test/uk/gov/hmrc/frontendtemplateprovider/GovUkTemplateRendererControllerSpec.scala

```diff
@@ -53,6 +64,25 @@ class GovUkTemplateRendererControllerSpec  extends AnyWordSpec with Matchers wit
       mimetype must include("text/html")
     }
 
+    "contain the dev template in the result" in {
+
+      when(config.environment).thenReturn("dev")
+
+      val result = sut.serveMustacheTemplate()(fakeRequest)
+      val bodyText = contentAsString(result)
+
+      bodyText must include("<a id=\"feedback-link\" href=\"http://localhost:9250/contact/beta-feedback-unauthenticated?service={{feedbackIdentifier}}\" data-sso=\"false\" data-journey-click=\"other-global:Click:Feedback\">")
```

Comment:
Think this could just be `include(http://localhost:9250)`

Reply:
Done